Update setup.md (#6978)
segmentio/segment-docs: src/connections/reverse-etl/setup.md

1. Navigate to **Connections > Sources** and select the **Reverse ETL** tab in the Segment app.
2. Click **+ Add Reverse ETL source**.
3. Select the source you want to add. 

> 4. Follow the corresponding guide to setup the required permissions for your Reverse ETL source:

Reviewer: ```suggestion
4. Follow the corresponding guide to set up the required permissions for your Reverse ETL source:
```

  - [Azure Reverse ETL setup guide](/docs/connections/reverse-etl/reverse-etl-source-setup-guides/azure-setup)
  - [BigQuery Reverse ETL setup guide](/docs/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup)
  - [Databricks Reverse ETL setup guide](/docs/connections/reverse-etl/reverse-etl-source-setup-guides/databricks-setup)
  - [Postgres Reverse ETL setup guide](/docs/connections/reverse-etl/reverse-etl-source-setup-guides/postgres-setup)
  - [Redshift Reverse ETL setup guide](/docs/connections/reverse-etl/reverse-etl-source-setup-guides/redshift-setup)
  - [Snowflake Reverse ETL setup guide](/docs/connections/reverse-etl/reverse-etl-source-setup-guides/snowflake-setup)

## Step 2: Add a model

> Models define sets of data you want to sync to your Reverse ETL destinations. A source can have multiple models. Segment supports SQL models and dbt models.

Author: Hyperlink to respective headers for SQL & dbt

> Models define sets of data you want to sync to your Reverse ETL destinations. A source can have multiple models. Segment supports SQL models and dbt models.

Reviewer: ```suggestion
Models define sets of data you want to sync to your Reverse ETL destinations. A source can have multiple models. Segment supports [SQL models](/docs/connections/reverse-etl/setup/#step-4-create-mappings) and [dbt models](/docs/segment-app/extensions/dbt/).
```

### SQL editor
1. Navigate to **Connections > Sources** and select the **Reverse ETL** tab. Select your source and click **Add Model**.
2. Click **SQL Editor** as your modeling method. (Segment will add more modeling methods in the future.)
3. Enter the SQL query that’ll define your model. Your model is used to map data to your Reverse ETL destination(s).
4. Choose a column to use as the unique identifier for each record in the **Unique Identifier column** field.

>     * The Unique Identifier should be a column with unique values per record to ensure checkpointing works as expected, such as a primary key. This column is used to detect new, updated, and deleted records.

Reviewer: ```suggestion
    * The Unique Identifier should be a column with unique values per record to ensure checkpointing works as expected, like a primary key. This column is used to detect new, updated, and deleted records.
```

6. Click **Next**.
7. Enter your **Model Name**.
8. Click **Create Model**.

### dbt model
Use Segment's dbt extension to centralize model management and versioning. Users who set up a BigQuery, Databricks, Postgres, Redshift, or Snowflake source can use Segment's [dbt extension](/docs/segment-app/extensions/dbt/) to centralize model management and versioning, reduce redundancies, and run CI checks to prevent breaking changes. 

## Step 3: Add a destination
In Reverse ETL, destinations are the business tools or apps you use that Segment syncs the data from your warehouse to. A model can have multiple destinations.

Refer to the [Reverse ETL catalog](/docs/connections/reverse-etl/reverse-etl-catalog/) to view the supported actions destinations. Reverse ETL supports to unique destinations:
- **[Segment Connections Destination](/docs/connections/reverse-etl/reverse-etl-catalog/#segment-connections-destination)**: Send warehouse data back into Segment to leverage your existing mappings or access non-actions destinations in the Connections catalog.

> - **[Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/)**: Engage users can use this destinaton to create and update [Profiles](/docs/unify/) that can then be accessed through [Profile API](/docs/unify/profile-api/) and activated within [Twilio Engage](/docs/engage). 

Reviewer: ```suggestion
- **[Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/)**: Engage Premier Subscriptions users can use Reverse ETL to sync subscription data from their warehouses to destinations.
```

Reviewer: This is what's in the RETL catalog page!

> - **[Segment Profiles Destination](/docs/connections/destinations/catalog/actions-segment-profiles/)**: Engage users can use this destinaton to create and update [Profiles](/docs/unify/) that can then be accessed through [Profile API](/docs/unify/profile-api/) and activated within [Twilio Engage](/docs/engage). 

Author: Could you rev this definition? Or copy what we provide in the public docs / internal docs?

4. Select the Reverse ETL source you want to connect the destination to.
5. Enter the **Destination name** and click **Create Destination**.
6. Enter the required information on the **Settings** tab of the destination.

> 7. Navigate to the destination settings tab and enable the destination. Note: If the destination is disabled, then Segment won't be able to start a sync.

Reviewer: ```suggestion
7. Navigate to the destination settings tab and enable the destination. If the destination is disabled, then Segment won't be able to start a sync.
```

4. Select the model to sync from.
5. Select the **Action** you want to sync and click **Next**.
      * Actions determine the information sent to the destination. The list of Actions will be unique to each destination.

> 6. Add the mapping name. The mapping name will default to the Action's name (e.g. 'Track Event'), but can be customized to allow you to identify the mapping across others.

Reviewer: We avoid using Latin phrases (e.g., i.e., etc.) in the docs where possible.

```suggestion
6. Add a name for your mapping. The mapping name defaults to the Action's name (for example, 'Track Event'), but can be customized to allow you to identify the mapping across others.
```

7. In the **Select record to map and send** section, select which records to send to your destination after Segment completes extracting data based on your model. You can choose from:
      * Added records
      * Updated records
      * Added or updated records
      * Deleted records
8. Select a test record to preview the fields that you can map to your destination in the **Add test record** field.

> 9. Select how often you want the schedule to sync in **Schedule configuration**.

Reviewer: ```suggestion
9. Select how often you want Segment to sync your data under **Schedule configuration**.
```

>     * **Interval**: Extractions perform based on a selected time cycle. You can choose from: 15 minutes, 30 minutes, 1 hour, 2 hours, 4 hours, 6 hours, 8 hours, 12 hours, 1 day.

Reviewer: ```suggestion
    * **Interval**: Extractions perform based on a selected time cycle. Select one of the following options: 15 minutes, 30 minutes, 1 hour, 2 hours, 4 hours, 6 hours, 8 hours, 12 hours, 1 day.
```
